### PR TITLE
German localization

### PR DESCRIPTION
**Pull request by justdave**
@chaosfreak93 Split off from #26 

Cherry-picked the patch from there, attribution intact.

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <string name="action_addserver">Server hinzufügen</string>
+    <string name="action_deleteserver">Server(s) löschen</string>
+    <string name="action_editserver">Server bearbeiten</string>
+    <string name="action_about">Über diese App</string>
+    <string name="action_help">Hilfe</string>
+    <string name="action_privacy">Privacy</string>
+    <string name="icon_description">Server Bild</string>
+    <string name="edit_server_name">Server Namen eingeben:</string>
+    <string name="edit_server_name_hint">Mein Minecraft Server</string>
+    <string name="edit_server_address">Server Addresse eingeben:</string>
```
Comment (justdave) [inline app/src/main/res/values-de/strings.xml:16]: The spellcheck in Android Studio is complaining that this should be 'Adresse', is that correct?

**Comment by justdave** [comment]
Lint is complaining about a bunch of things on this branch, but they're all fixed in #28, so if that lands first, this will be fine.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    <string name="app_name">MCStatus</string>
+    <string name="about_version">Version %1$s (Build %2$d)</string>
+    <string name="action_refresh">Neuladen</string>
+    <string name="action_addserver">Server hinzufügen</string>
+    <string name="action_deleteserver">Server(s) löschen</string>
+    <string name="action_editserver">Server bearbeiten</string>
+    <string name="action_about">Über diese App</string>
+    <string name="action_help">Hilfe</string>
+    <string name="action_privacy">Privacy</string>
```
Comment (justdave) [inline app/src/main/res/values-de/strings.xml:12]: @chaosfreak93 A German student I know suggests this one might want to be "Datenschutzerklärung", but she's a student and not a native speaker, so I'll defer to the native speaker on that.

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (chaosfreak93) [reply app/src/main/res/values-de/strings.xml:16]: Yes, my bad 'Adresse' is right

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (chaosfreak93) [reply app/src/main/res/values-de/strings.xml:12]: I think 'Datenschutz' sounds better

File: app/src/main/res/raw-de/privacy.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<b>MCStatus Privacy Info</b><br>
```
Comment (chaosfreak93) [inline app/src/main/res/raw-de/privacy.txt:1]: MCStatus Datenschutzerklärung

File: app/src/main/res/raw-de/about_info.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Sie haben einen Fehler gefunden oder haben eine Idee für die App<br>
+<b>https://github.com/justdave/MCStatus/issues</b><br>
+<br>
+Copyright 2014-2020 David D. Miller<br>
+Copyright 2020 Rick van Schijndel<br>
+Copyright 2020 Yan-Luca Lentzen<br>
```
Comment (chaosfreak93) [inline app/src/main/res/raw-de/about_info.txt:6]: Copyright 2020-2021 Yan-Luca Lentzen

**Pull request merged**